content/courses/program-optimization/program-configuration.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Program Configuration
+title: Admin configuration
```

Comment:
@wuuer this is great, sorry it's the last big PR and I've just gotten around to reviewing it.